mama/c_cpp/src/c/mama/msg.h

```diff
@@ -62,7 +64,8 @@ typedef enum mamaPayloadType_
     MAMA_PAYLOAD_EXEGY      = 'X',
 
 /* Unknown Payload */
-    MAMA_PAYLOAD_UNKNOWN    = 'U'
+    MAMA_PAYLOAD_UNKNOWN    = 'U',
+    MAMA_PAYLOAD_NULL       = '\0'
```

Comment:
As discussed the issue is not readibility - it is more the fact that this is a new enum value for an enum we're trying to deprecate - hence saying this should be a macro value. I suggested "_ID_*" to differentiate between macro-defined payload types and enum payload types.